benchmark/util/inspect.js

```diff
@@ -3,14 +3,43 @@ var util = require('util');
 
 var common = require('../common.js');
 
-var bench = common.createBenchmark(main, {n: [5e6]});
+var bench = common.createBenchmark(main, {n: [3e6]});
 
 function main(conf) {
   var n = conf.n | 0;
 
   bench.start();
   for (var i = 0; i < n; i += 1) {
-    util.inspect({a: 'a', b: 'b', c: 'c', d: 'd'});
+    util.inspect({
+      a: undefined,
+      b: null,
+      c: [
+        // test for empty array index
+        , // eslint-disable-line comma-style
```

Comment:
Updated. The current ESLint setting allows this style: https://github.com/shinnn/node/blob/144a2e57f6498d0fd6eb83190c4ab059e45da3cc/benchmark/util/inspect.js#L16